app/response/utils.py

```diff
@@ -1360,9 +1360,23 @@ def _denial_email_handler(request_id, data, page, agency_name, email_template):
 
     :return: the HTML of the rendered template of a closing
     """
+    point_of_contact = data.get('point_of_contact', None)
```

Comment:
This should probably be a function. We use it in all of the letter handlers and will probably end up using it in all of the email handlers as well, right?